.changeset/metal-fireants-switch.md

```diff
@@ -0,0 +1,55 @@
+---
+"@spectrum-css/dialog": major
+"@spectrum-css/modal": patch
+"@spectrum-css/underlay": patch
+---
+
+Spectrum 2 Dialog Migrations
+
+To separate itself from another component named "alert dialog," dialog was renamed to "standard dialog." The Divider component is no longer supported in S2 standard dialogs. Checkbox and text-only options are available in the footer content area. In addition, some new and updated tokens are in place to update the standard dialog corner rounding, font treatments, spacing/padding, and maximum/minimum widths. There were several missing elements of the standard dialog implemented, including optional header and footer content, hero/cover images, and supporting fullscreen layouts.
+
+Mod properties are either "new," have been renamed to reflect the "standard dialog" language, or were removed:
+
+_New Mods_
```

Comment:
I need to add the takeover dialog & window-to-edge tokens to the changeset.

Reply:
I could also add a note about completely removing the `--small`, `--medium`, `--large` class names in favor of t-shirt sizes. I also corrected the spelling of the `is-dismissible` class.